gm4_area_markers/data/area_markers/functions/marker/process.mcfunction

```diff
@@ -0,0 +1,10 @@
+#called from main
+#@s = area marker
+
+#make message apear
+execute at @s[tag=gm4_am_tier_1] positioned ~-24 1 ~-24 run title @a[dx=48, dy=299, dz=48] actionbar [{"translate":"%1$s%3427655$s","with":["Area: ",{"translate":"text.gm4.area_markers.area"}],"color":"gold"},{"selector":"@s","color":"white"}]

```

Comment:
There will be conflicts here if multiple markers overlap.

Reply:
Generally the newest one takes over but it could just be coincidence.